modules/ROOT/pages/datasense-perceptive-flow-design-concept.adoc

```diff
@@ -6,45 +6,54 @@ endif::[]
 
 Perceptive Flow Design is an Anypoint Studio functionality that lets you visualize the mapping of data from one data format and structure to another.
 
-Consider a flow in which you included an Anypoint Connector that's preceded or followed by a Transform Message component. Mule Runtime uses the working connection to the resource to retrieve metadata about the payload and properties and feeds this data into DataWeave, providing the expected input or output. This avoids you the trouble of manually inspecting the Mule Event's structure yourself.
+== Purpose of Perceptive Flow Design
 
-For example, imagine that you need to connect your organization's Salesforce accounts with Twitter to publicize specific performance indicators.
+Mule 4 introduces a prescriptive way to build reliable and reusable flow artifacts and apps. You can build, for example, more reliable applications by achieving earlier detection of errors during design time. To make this feature possible without changing runtime semantics, Mule 4 uses metadata. Metadata should act as a prescriptive guidance to build more reliable and reusable applications.
 
-By dropping two connectors on your Studio canvas – Salesforce and Twitter – and configuring them to connect to your organization's accounts, you can then drop a Transform Message component between the connectors and inspect its Mule Properties View to find that Mule Runtime has intelligently captured the data type and structure information from each  provider, and prescribed the input and output for your data mapping.
+Consider a flow in which you included an Anypoint Connector that's preceded or followed by a Transform Message component. The Mule runtime engine uses the working connection to the resource to retrieve metadata about the payload and properties, and feeds this data into DataWeave providing the expected input or output, and avoiding you the trouble to manually inspect the Mule event's structure.
```

Comment:
"...input or output, and avoiding you..." I think the **and** needs to be removed: "...input or output, avoiding you... "